shims-vue.d.ts

```diff
@@ -0,0 +1,16 @@
+// /* eslint-disable */
```

Comment:
Can this part be deleted? Since it's just commented on

Reply:
Ok, thanks! I forgot to delete. I'll delete here :)